.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: Bug Report
+description: File a bug report to help us improve
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: "# Please search the issue tracker for existing bug reports before submitting your own."
+  - type: input
+    id: system-information
+    attributes:
+      label: System Information
+      description: |
+        - The operating system you use to run LMMS.
+        - When relevant, also include your hardware information.
+      placeholder: ex. Fedora Linux 39, KDE Plasma 5.27.10 - 13th Gen Intel® Core™ i9-13950HX, 32GB RAM
+    validations:
+      required: true
+  - type: input
+    id: affected-version
+    attributes:
+      label: LMMS Version(s)
+      description: |
+        - The version of LMMS affected by the bug.
+        - Can be an official version number, nightly release identifier, or commit hash.
+        - The version number can be found under the Help > About menu.
+      placeholder: ex. 1.2.2, 1.3.0-alpha.1.518+gdd53bec31, 2d185df
+    validations:
+      required: true
+  - type: input
+    id: working-version
+    attributes:
+      label: Most Recent Working Version
+      description: |
+        - If there is a previous version of LMMS that did not exhibit the bug, please include it here.
+        - Using `git bisect` to determine exactly where the regression occurred will make it easier for LMMS developers to find and fix the bug.
+      placeholder: ex. 1.2.2, 1.3.0-alpha.1.518+gdd53bec31, 2d185df
+    validations:
+      required: false
+  - type: textarea
+    id: bug-summary
+    attributes:
+      label: Bug Summary
+      description: Briefly describe the bug and what LMMS did that was not expected.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected Behaviour
+      description: Describe what should have happened and how LMMS should have behaved.
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps To Reproduce
+      description: |
+        - Describe the minimum steps one can follow to reproduce this bug for themself.
+        - If you included an MRP below, you can describe here how it should be used.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: |
+        - Please copy and paste any relevant log output where instructed.
+        - LMMS log output can be obtained by running it via the console.
```

Comment:
LMMS doesn't produce console output by default on Windows. Also, "where instructed" doesn't make sense to me here - are we guiding users through this form elsewhere? I would have thought this form itself would usually constitute the instructions.

Reply:
I had been referring to the comment inside the HTML block when I said "where instructed", but I think that it's actually clear enough by itself and what I said doesn't make sense, you're right.

I took out the note about console output entirely and left it to the user, but I'm not sure if that's the right approach either.